### DISCUSSION TITLE
Toast Personalisation

### DISCUSSION THREAD

**Discussion by roberthuarcaya** [jpmorganchase/salt-ds#3246] [Q&A] [answered]
Hello, 
I am using the toast below 
https://www.saltdesignsystem.com/salt/components/toast/examples#success

I want the toast to appear for only a few seconds and then go away. Besides, it should be over the main website; I do not want it to be created like an actual element on my website. How can I achieve those 2 features? 

Note: how can I access github using my jpmc email?

**Comment by origami-z** [comment, marked as answer]
You need to manage this yourself, have a state to render the component (possibly in absolute position), then remove after a few seconds.

For internal Q&A: https://go/ask-salt
